script.js

```diff
@@ -0,0 +1,59 @@
+var choice = ["Rock", "Paper", "Scissors"];  
+choice.forEach(Function1);
+function Function1(){
+var btn = document.createElement("BUTTON");   
+btn.innerHTML = "index";                                
+document.getElementById("div1").appendChild(btn);
+}
+var player = prompt("Let's GOoOOOoOo");
+if (! player) {
+    
+    document.alret("should chose one of these optiones  " );
+} else {
+    
+    document.write("<p> player choice:" + " " + player+ "</p>" );
+}
+function computerChoice (){
+var computer= choice[Math.floor(Math.random() * choice.length)];
+
+        // Display computer choice
+        document.write("<p>Computer choice:" + " " + computer+ "</p>");
+}
+var win = function(computerChoice,player){
+    if (computer === player) {
+        return "It's a tie!";
+    }
+    if (computer === "rock") {
+        if (player === "scissors") {
+            // rock wins
+            return "You win!";
```

Comment:
it's suppose to return **"you lost"** for the user :thinking: 
![nononono](https://user-images.githubusercontent.com/57558867/98463248-a0cb9280-21c2-11eb-99f5-a3098e7d9368.png)